proxy/hdrs/HTTP.h

```diff
@@ -159,7 +159,8 @@ enum SquidLogCode {
   SQUID_LOG_ERR_FUTURE_1              = 'I',
   SQUID_LOG_ERR_CLIENT_READ_ERROR     = 'J', // Client side abort logging
   SQUID_LOG_ERR_LOOP_DETECTED         = 'K', // Loop or cycle detected, request came back to this server
-  SQUID_LOG_ERR_UNKNOWN               = 'Z'
+  SQUID_LOG_ERR_UNKNOWN               = 'Z',
+  SQUID_LOG_TCP_CF_HIT                = 'F' // Collapsed forwarding HIT also known as Read while write hit
```

Comment:
[nitpicking] why only this is named `CF_HIT`? For consistency with something?

Reply:
Yes, it is named according to squid format. 
https://wiki.squid-cache.org/SquidFaq/SquidLogs

Reply:
CF --> "Collapsed Forwarding"

Reply:
Looks like this list is sorted alphabetically by value , the new option should be moved up

Reply:
@randall yeah, i will update that.